.changeset/chatty-balloons-impress.md

```diff
@@ -0,0 +1,44 @@
+---
+"wrangler": minor
+---
+
+feat: Support runtime-agnostic polyfills
+
+Previously, Wrangler treated any imports of `node:*` modules as build-time errors (unless one of the two Node.JS compatibility modes was enabled). This is sometimes overly aggressive, since those imports are often not hit at runtime (for instance, it was impossible to write a library that worked across Node.JS and Workers, using Node packages only when running in Node). Here's an example of a function that would cause Wrangler to fail to build:
```

Comment:
```suggestion
Previously, Wrangler treated any imports of `node:*` modules as build-time errors (unless one of the two Node.js compatibility modes was enabled). This is sometimes overly aggressive, since those imports are often not hit at runtime (for instance, it was impossible to write a library that worked across Node.JS and Workers, using Node packages only when running in Node). Here's an example of a function that would cause Wrangler to fail to build:
```

Reply:
Just a nit... it's `Node.js` ... (lowercase `js`)